api_handler.go

```diff
@@ -42,9 +42,63 @@ type withSetBundle interface {
 	SetBundle(bundle *apiBundle)
 }
 
+type statusResponse struct {
```

Comment:
A lot of endpoints return `{status: "ok"}`, so I made a common struct that they can share so we don't have to write a new response struct for each. `statusResponseOK` below is also a shortcut for making an OK response.